product_brand/__openerp__.py

```diff
@@ -0,0 +1,58 @@
+# -*- encoding: utf-8 -*-
+###############################################################################
+# #                                                                           #
+# product_brand for Odoo #                                                    #
+# Copyright (C) 2009 NetAndCo (<http://www.netandco.net>). #                  #
+# Copyright (C) 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com> #  #
+# Copyright (C) 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com> #   #
+# Contributors                                                                #
+# Mathieu Lemercier, mathieu@netandco.net, #                                  #
+# Franck Bret, franck@netandco.net #                                          #
+# Seraphine Lantible, s.lantible@gmail.com, http://www.prisnet.ch             #
+# #                                                                           # 
```

Comment:
@vrms You are almost done for pep8. You just have an extra space on this line as stated in travis:

./product_brand/__openerp__.py:12:80: W291 trailing whitespace